packages/app-store/discord/config.json

```diff
@@ -0,0 +1,26 @@
+{
+  "/*": "Don't modify slug - If required, do it using cli edit command",
+  "name": "Discord",
+  "slug": "discord",
+  "type": "discord_video",
+  "imageSrc": "/api/app-store/discord/icon.svg",
+  "logo": "/api/app-store/discord/icon.svg",
```

Comment:
`imageSrc` is deprecated. We should use `logo` only everywhere. Currently it seems like some places are still using `imageSrc` which doesn't support specifying paths relative to `static` dir
<img width="754" alt="Screenshot 2023-04-05 at 5 04 59 PM" src="https://user-images.githubusercontent.com/1780212/230068749-6eab3963-7ea2-4d7e-8310-16961d11461e.png">

Reply:
Created a ticket and assigned to myself.